### PR TITLE
[MM-31045] Fix Error when adding group on channel configuration screen

### PR DESCRIPTION
**Pull request by hahmadia**
#### Summary

Fix a small bug in typescript migration that was causing lots of issues

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-31045

**Note to QA**
Ensure that when you click edit on a channel, the channel loads fine and that all the functions on that page work (do not throw a error or white screen of death)

### CODE REVIEW

File: components/admin_console/team_channel_settings/channel/details/channel_groups.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ interface ChannelGroupsProps {
     isDisabled?: boolean;
 }
 
-export const ChannelGroups: React.SFC<ChannelGroupsProps> = (props: ChannelGroupsProps): JSX.Element => {
+export const ChannelGroups: React.FunctionComponent<ChannelGroupsProps> = (props: ChannelGroupsProps): JSX.Element => {
```
Comment (hahmadia) [inline components/admin_console/team_channel_settings/channel/details/channel_groups.tsx:29]: This was a un-related deprecated function fix. (SFC is deprecated and need to use FunctionComponent)

**Comment by hahmadia** [comment]
Couldn't really think of a way to add a test as there was already a big console error saying that `Channel ID is marked as required but received undefined`

**Review by mkraft — APPROVED** [review]

**Review by sbishel — APPROVED** [review]

**Review by furqanmlk — CHANGES_REQUESTED** [review]
I am getting `NaN Members` instead of `0 Member`

image

**Review by furqanmlk — APPROVED** [review]
Could not reproduce this issue mentioned above.
During Cypress test case, group was deleted using API while UI is visible, so it is not normal behaviour. 
All test cases passed

**Pull request merged**